examples/pymtbl_make_random_table_2.py

```diff
@@ -11,9 +24,16 @@
 report_interval = 100000
 megabyte = 1048576.0
 
+try:
+    maxint = sys.maxint  # py2
+except:
```

Comment:
Should be `except AttributeError:` instead

Reply:
Fixed. Thanks